### PR TITLE
Set Fuse getattr size to output stream bytes written

### LINKED ISSUE #14713: Show correct bytes written in Fuse getattr
**Alluxio Version:**
Master

**Describe the bug**
Alluxio master only updates file status length after file completed.
But in Alluxio Fuse, sometimes applications need to known the bytes written by calling getattr.
At least in the same Alluxio Fuse which file is written to, we should support getting the bytes written info in getattr

**To Reproduce**
Steps to reproduce the behavior (as minimally and precisely as possible)

**Expected behavior**
Same Fuse mount point, getattr() can show the correct bytes written even if file is not completed

**Urgency**
Describe the impact and urgency of the bug.

**Are you planning to fix it**
Please indicate if you are already working on a PR. 

**Additional context**

### PR DESCRIPTION
**Pull request by LuQQiu**
### What changes are proposed in this pull request?

Fixes #14713

### Why are the changes needed?
Show correct bytes written in Fuse.getattr() when file is writing to that fuse mount point

### CODE REVIEW

**Comment by LuQQiu** [comment]
@ssz1997 @yyongycy PTAL, thanks!

File: integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java [hunk 1]
```diff
@@ -239,16 +239,26 @@ private int getattrInternal(String path, FileStat stat) {
       } else {
         status = mFileSystem.getStatus(uri);
       }
+      long size = status.getLength();
```
Comment (yyongycy) [inline integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java:242]: The purpose to get the size here is to make sure the size is gotten at the beginning? Size could be changed during the gap between line 244 and 259?

Comment (yyongycy) [reply integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java:242]: If it could be changed, why not return the value after it is done?

Comment (yyongycy) [reply integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java:242]: Does this change mean return the size even if the file is not finished yet?

Comment (LuQQiu) [reply integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java:242]: Alluxio will not update the file length until the file is completed. So when the file is being written, Alluxio Master only have the file metadata showing the file length is 0.
so at line 242, the file length is zero.

This is fine in many cases but also may cause some applications to error out.
e.g. Some applications calls create() - write() - getattr() - write().
If the getattr() between the two writes show written size 0, the application may error out

Comment (LuQQiu) [reply integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java:242]: we block waiting for file to be completed when the file is not written by this FUSE.
we cannot block waiting for file is completed when the file is being written by the same FUSE, then the second write() will never be called.

**Comment by LuQQiu** [comment]
@maobaolong Please also take a look, thanks

**Review by yyongycy — APPROVED** [review]
Then LGTM,

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**